main.js

```diff
@@ -1292,49 +1296,6 @@ require(['jquery', 'underscore', 'utils', 'video', 'soundchip', 'ddnoise', 'debu
             ddNoise.mute();
         }
 
-        (function () {
-            const $cubMonitor = $("#cub-monitor");
-            var isFullscreen = false;
-            var cubOrigHeight = $cubMonitor.height();
-            var cubToScreenHeightRatio = $screen.height() / cubOrigHeight;
-            var cubOrigWidth = $cubMonitor.width();
-            var cubToScreenWidthRatio = $screen.width() / cubOrigWidth;
-            var navbarHeight = $("#header-bar").height();
-            const desiredAspectRatio = cubOrigWidth / cubOrigHeight;
-            const minWidth = cubOrigWidth / 4;
-            const minHeight = cubOrigHeight / 4;
-            const borderReservedSize = 100;
-            const bottomReservedSize = 100;
-
-            function resizeTv() {
```

Comment:
Losing this functionality means the screen area doesn't scale to fit the window in non-fullscreen mode. That was the main reason for most of this code: it means on my silly giant monitor I don't have a vast open black space with a small BBC micro screen in it :)  (this from my smaller laptop screen...
![Screenshot from 2019-08-26 07-55-14](https://user-images.githubusercontent.com/633973/63692338-08243f00-c7d7-11e9-9ff0-2489a1f0f80e.png) - normally looks like this: 
![Screenshot from 2019-08-26 07-57-17](https://user-images.githubusercontent.com/633973/63692375-25590d80-c7d7-11e9-8cbf-ba361788a037.png) )